history/view.qtpl

```diff
@@ -13,6 +13,24 @@ HyphaeLinksHTML returns a comma-separated list of hyphae that were affected by t
 {% endstripspace %}
 {% endfunc %}
 
+
+HyphaeDiffsHTML returns a comma-separated list of diffs links of current revision for every affected file as HTML string.
+{% func (rev Revision) HyphaeDiffsHTML() %}
+    {% code entries := rev.hyphaeAffected() %}
+{% stripspace %}
+    {% if len(entries) == 1 %}
+        <a href="/primitive-diff/{%s rev.Hash %}/{%s entries[0] %}">{%s rev.Hash %}</a>
+    {% else %}
+        {% for i, hyphaName := range entries %}
+            {% if i > 0 %}
+                <span aria-hidden="true">, </span>
+            {% endif %}
+                <a href="/primitive-diff/{%s rev.Hash %}/{%s hyphaName %}">{%s rev.Hash %}({%s hyphaName %})</a>
```

Comment:
I think you should have a space before the opening parenthesis.

hashhash (hypha)

Reply:
Inserted non-breaking one